src/test/java/org/scijava/display/DisplayTest.java

```diff
@@ -132,4 +132,14 @@ public void testText() {
 		assertEquals(value, result);
 	}
 
+	@Test
+	public void testMultipleDisplaysPriorityMatch() {
+		final Context context = new Context(DisplayService.class);
+		final DisplayService displayService =
+				context.getService(DisplayService.class);
+		final String name = "Text";
+		final String value = "Hello";
+		Display<?> display = displayService.createDisplay(name, value);
+		assertEquals(CustomTextDisplay.class, display.getClass());
```

Comment:
In addition to asserting that `CustomTextDisplay` gets chosen, can we also test that we actually do get *multiple* candidate `Display`s? Otherwise the name of this method would be misleading, no?